src/couch_replicator_scheduler.erl

```diff
@@ -30,13 +30,14 @@
 -export([handle_config_change/5, handle_config_terminate/3]).
 
 %% types
--type event_type() :: started | stopped | crashed.
+-type event_type() :: started | stopped | {crashed, any()}.
```

Comment:
For 1) introspection and 2) the ability to make scheduling decisions based on specific errors. Thoughts?